goSrc/src/vmCaller/evm/native_contract.go

```diff
@@ -308,6 +308,130 @@ var (
 			F:       createRole,
 		},
 	)
+		QueryServiceContract = native.New().MustContract("QueryServiceContract",
+		`* acmstate.ReaderWriter for bridging EVM state and Iroha state.
+			* @dev This interface describes the functions exposed by the native service contracts layer in burrow.
```

Comment:
```suggestion
			* @dev This interface describes the functions exposed by the native service contracts layer in Hyperledger Burrow.
```

Reply:
this description is copied from original iroha, not made by me